crates/sui-storage/src/http_key_value_store.rs

```diff
@@ -52,9 +52,8 @@ pub fn encoded_tagged_key(key: &TaggedKey) -> String {
     base64_url::encode(&bytes)
 }
 
-pub fn encode_object_key(object_id: &ObjectID, version: &VersionNumber) -> String {
-    let bytes =
-        bcs::to_bytes(&ObjectKey(*object_id, *version)).expect("failed to serialize object key");
+pub fn encode_object_key(object_key: &ObjectKey) -> String {
+    let bytes = bcs::to_bytes(object_key).expect("failed to serialize object key");
```

Comment:
was this change just a cleanup, or is there another reason for changing the inputs here?

Reply:
Small refactor to make things slightly more efficient/convenient